src/commands/lambda/renderers/flare-renderer.ts

```diff
@@ -11,5 +11,25 @@ import {dryRunTag} from './common-renderer'
 export const renderLambdaFlareHeader = (isDryRun: boolean) => {
   const prefix = isDryRun ? `${dryRunTag} ` : ''
 
-  return `\n${prefix}🐶 Generating Lambda flare to send your configuration to Datadog\n`
+  return `\n${prefix}🐶 Generating Lambda flare to send your configuration to Datadog.\n`
 }
+
+/**
+ * @param error an error message
+ * @returns the provided error prefixed by {@link errorTag}.
+ *
+ * ```txt
+ * [Error] The provided error goes here!
+ * ```
+ */
+export const renderError = (error: string) => `\n${errorTag} ${error}\n`
```

Comment:
This seems to be used in https://github.com/DataDog/datadog-ci/blob/df721c1b57bd2925bd008ab9f15d4cc50e4d7ec5/src/commands/lambda/renderers/instrument-uninstrument-renderer.ts#L144

Can we move them into `common-renderer` to reuse the code?